programs/ssl/ssl_client2.c

```diff
@@ -982,6 +995,20 @@ int main( int argc, char *argv[] )
             if( opt.debug_level < 0 || opt.debug_level > 65535 )
                 goto usage;
         }
+#if defined(MBEDTLS_VERSION_C)
+        else if( strcmp( p, "build_version" ) == 0 )
+        {
+            if( strcmp( q, "1" ) == 0 )
+            {
+                char version_str[10];
```

Comment:
Duplicated coce between here and below. This could be moved into a (possibly inline) function.

Reply:
As the code has been modified as a single `mbedtls_printf` to print build version info directly. I think it's not needed to be a function. How do you think?